core/templates/dev/head/pages/exploration_player/PlaythroughService.js

```diff
@@ -49,6 +49,17 @@ oppia.factory('PlaythroughService', [
     var cycleIdentifier = {};
     var visitedStates = [];
 
+    /**
+     * A quit action is recorded every time a MaybeQuit event is encountered.
+     * This function extracts the old quit action out of the playthrough
```

Comment:
This doc can be removed since the function is now self-explanatory.

Reply:
Done.